components/clp-package-utils/clp_package_utils/controller.py

```
@@ -796,6 +728,73 @@ def _set_up_env(self) -> None:
+        """
+        Starts CLP's components using Docker Compose.
+
+        :raise: Propagates `check_docker_dependencies`'s exceptions.
+        :raise: Propagates `subprocess.run`'s exceptions.
+        """
+        check_docker_dependencies(
+            should_compose_project_be_running=False, project_name=self._project_name
+        )
+
+        deployment_type = self._clp_config.get_deployment_type()
+        logger.info(f"Starting CLP using Docker Compose ({deployment_type} deployment)...")
```

Ruff (0.14.1)

Logging statement uses f-string

(G004)